#! /bin/sh
## 02_dhcp_probe.8.diff by Laurent Guignard <lguignard.debian@gmail.com>
##
## DP: Change a syntax problem in groff file for hyphenation problem
## Note:
##   Submitted to upstream programmer Irwin Tillman<irwin@princeton.edu>.
##       This patch will be applied in the next version of the upstream code.


@DPATCH@
Index: trunk/doc/dhcp_probe.8
===================================================================
--- trunk.orig/doc/dhcp_probe.8	2010-03-15 21:55:49.000000000 +0100
+++ trunk/doc/dhcp_probe.8	2010-03-15 21:59:03.000000000 +0100
@@ -380,15 +380,15 @@
 file,
 that program is executed, with the following required options:
 .nf
-   -p the name of the calling program (e.g. dhcp_probe)
-   -I the name of the interface on which the unexpected response packet was received
-   -i the IP source address of the packet
-   -m and the Ethernet source address of the packet
+   \-p the name of the calling program (e.g. dhcp_probe)
+   \-I the name of the interface on which the unexpected response packet was received
+   \-i the IP source address of the packet
+   \-m and the Ethernet source address of the packet
 .fi
 If the response packet's yiaddr is non-zero and falls within a "Lease Networks of Concern",
 the following optional options are also passed:
 .nf
-   -y the non-zero yiaddr value
+   \-y the non-zero yiaddr value
 .fi
 (We do not wait for the
 .I alert_program_name2
@@ -874,11 +874,11 @@
 Configuration file read by the program.
 See
 .BR dhcp_probe.cf (5).
-The name of this file can be overriden by a command-line option.
+The name of this file can be overridden by a command-line option.
 .TP
 .B /etc/dhcp_probe.pid
 Contains the program's processid.
-The name of this file can be overriden by a command-line option.
+The name of this file can be overridden by a command-line option.
 .SH LIMITATIONS
 dhcp_probe 
 is not guaranteed to locate all unknown DHCP and BootP
